csharp/BUILD

```diff
@@ -52,9 +51,96 @@ csharp_library(
         "//csharp/Connection:connection",
         "//csharp:typedb_driver_pinvoke",
     ],
-    visibility = ["//visibility:public"],
+    out = "TypeDB.Driver",
     target_frameworks = target_frameworks,
     targeting_packs = targeting_packs,
+    visibility = ["//visibility:public"],
+)
+
+
+swig_native_nuget_pack(
+    name = "driver-csharp-runtime-pack",
+    id = "TypeDB.Driver.Pinvoke",
+    libs = {
+        ":typedb_driver_pinvoke": "TypeDB.Driver.Pinvoke",
+    },
+    native_libs = ["typedb_driver"],
+    files = {
+        "README.md": "README.md",
+    },
+    target_framework = target_framework,
+    nuspec_template = "TypeDB.Driver.Pinvoke.nuspec",
+    visibility= ["//visibility:public"],
+    platforms = {
+        "@vaticle_bazel_distribution//platform:is_mac_arm64": "osx-arm64",
+        "@vaticle_bazel_distribution//platform:is_mac_x86_64": "osx-x64",
+        "@vaticle_bazel_distribution//platform:is_linux_arm64": "linux-arm64",
+        "@vaticle_bazel_distribution//platform:is_linux_x86_64": "linux-x64",
+        "@vaticle_bazel_distribution//platform:is_windows_x86_64": "win-x64",
+    },
+)
+
+
+nuget_push(
```

Comment:
Pushes the generated `.nuspec` and `.snuspec` files based on the `snapshot / release` cmd flag.